week2/solutions/ccbill_project/cal_min_pay.py

```diff
@@ -0,0 +1,30 @@
+'''
+Assignment: 
+For each month, calculate statements on the monthly payment and remaining balance.
+At the end of 12 months, print out the remaining balance. 
+Be sure to print out no more than two decimal digits of accuracy - so print
+Remaining balance: 813.41
+
+So your program only prints out one thing: the remaining 
+balance at the end of the year in the format:
+
+Monthly interest rate= (Annual interest rate) / 12.0
+Minimum monthly payment = (Minimum monthly payment rate) x (Previous balance)
+Monthly unpaid balance = (Previous balance) - (Minimum monthly payment)
+Updated balance each month = (Monthly unpaid balance) + (Monthly interest
+rate x Monthly unpaid balance)
+'''
+def cal_min_pay(balance, annual_interest_rate, monthly_payment_rate):
```

Comment:
This isn't a useful function name. What is `cal`? What is `pay`?

Can you find a better name for this function that better describes what it actually does?